sinks/api/http/endpoint.go

```diff
@@ -125,7 +132,7 @@ func updateSinkEndpoint(svc sinks.SinkService) endpoint.Endpoint {
 			return nil, err
 		}
 		var config types.Metadata
-		if req.Format != "" {
+		if req.Format == "yaml" {
```

Comment:
is this modification necessary again? or your branch is not updated with develop?